### PR TITLE
Docs: Update Installation Guide for Ubuntu

### LINKED ISSUE #732: Update Ubuntu install guide for 16.10 and 17.04 (and 17.10 tomorrow!)
The [Ubuntu install guide](/clearcontainers/runtime/blob/master/docs/ubuntu-installation-guide.md) needs to be tested on Ubuntu 16.10, 17.04 and (as of 19 October) 17.10), and the text updated to state it supports these releases.

(For 17.10 we'll need @gorozco1's help to update OBS).

Related #731.

### PR DESCRIPTION
**Pull request by GabyCT**
This will update the installation guide for ubuntu 16.10 and 17.04

Fixes #732

Signed-off-by: Gabriela Cervantes <gabriela.cervantes.tellez@intel.com>

### CODE REVIEW

**Comment by GabyCT** [comment]
@iphutch and @jodh-intel, I made a mistake in the previous pr could you just do the lgtm to this one please?

File: docs/ubuntu-installation-guide.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ For more information on installing Docker please refer to the
 
 ```
 $ sudo sh -c "echo 'deb http://download.opensuse.org/repositories/home:/clearcontainers:/clear-containers-3/xUbuntu_$(lsb_release -rs)/ /' >> /etc/apt/sources.list.d/clear-containers.list"
-$ curl -fsSL http://download.opensuse.org/repositories/home:/clearcontainers:/clear-containers-3/xUbuntu_$(lsb_release -rs)/Release.key | sudo apt-key add -
+$ wget -qO - http://download.opensuse.org/repositories/home:/clearcontainers:/clear-containers-3/xUbuntu_$(lsb_release -rs)/Release.key | sudo apt-key add -
 $ sudo -E apt-get update
 $ sudo -E apt-get -y install -y cc-runtime cc-proxy cc-shim
```
Comment (jodh-intel) [inline docs/ubuntu-installation-guide.md]: Sorry to be a pain, but could you remove the 2nd `-y` in this line in either the same commit or a new one (this was my fault I think! :)

File: docs/ubuntu-installation-guide.md [hunk 2]
```diff
@@ -5,14 +5,14 @@ Note:
...
+Clear Containers **3.0** is available for Ubuntu\* **16.04** , **16.10** and **17.04**.
 
 This step is only required in case Docker is not installed on the system.
 1. Install the latest version of Docker with the following commands:
 
 ```
-$ sudo -E apt-get -y install apt-transport-https ca-certificates curl software-properties-common
-$ curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo apt-key add -
+$ sudo -E apt-get -y install apt-transport-https ca-certificates wget software-properties-common
+$ wget -qO - https://download.docker.com/linux/ubuntu/gpg | sudo apt-key add -
```
Comment (jodh-intel) [inline docs/ubuntu-installation-guide.md]: Why are we switching to `wget` by the way?

Comment (GabyCT) [reply docs/ubuntu-installation-guide.md]: @jodh-intel there is an issue with curl with ubuntu 17.04 
$ curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo apt-key add -
curl: (35) Unknown SSL protocol error in connection to ...
gpg: no valid OpenPGP data found.

File: docs/ubuntu-installation-guide.md (hunk 1 above)
Comment (GabyCT) [reply docs/ubuntu-installation-guide.md]: @jodh-intel done :)

Comment (jodh-intel) [reply docs/ubuntu-installation-guide.md]: Thanks very much! ;)

File: docs/ubuntu-installation-guide.md (hunk 2 above)
Comment (jodh-intel) [reply docs/ubuntu-installation-guide.md]: It's working for me on a 17.04 box using this version of `curl`:

```
$ lsb_release -r                                                                
Release:        17.04
$ curl -fsSL https://download.docker.com/linux/ubuntu/gpg &>/dev/null && echo ok
ok
$ dpkg -l curl|grep ^ii|awk '{print $3}'                                        
7.52.1-4ubuntu1.2
```

Is you system fully up-to-date?

Comment (GabyCT) [reply docs/ubuntu-installation-guide.md]: @jodh-intel, yes
$ lsb_release -r
Release:        17.04
$ curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo apt-key add -
curl: (35) Unknown SSL protocol error in connection to proxy-chain.intel.com:911
gpg: no valid OpenPGP data found.
$ dpkg -l curl|grep ^ii|awk '{print $3}'
7.52.1-4ubuntu1.2

Comment (jodh-intel) [reply docs/ubuntu-installation-guide.md]: ok - looks like a proxy issue. `wget` it is then! ;)

**Comment by jodh-intel** [comment]
lgtm

File: docs/ubuntu-installation-guide.md (hunk 2 above)
Comment (GabyCT) [reply docs/ubuntu-installation-guide.md]: @jodh-intel :)

**Pull request merged**